elenacaseyroby/YoutubeMusicLib
# posts mult listens of same vid

# What

Posts multiple listens of the same video.
# Why

To track all user listens.
# Test
1. Go to http://localhost:5000/ and play a video that is already saved in the videos table.
2. Check the listens table to make sure it recorded the listen. 
3. If all looks good ~ merge it!
